nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/FeaturesRunner.java

```diff
@@ -287,18 +287,31 @@ public void evaluate() throws Throwable {
 
     }
 
-    protected class BeforeSetupStatement extends Statement {
+    protected abstract class MethodStatement extends Statement {
```

Comment:
perhaps no needed in the test context, but should we add a `@since ..`

Reply:
It's not strictly needed to add `@since` on protected fields, as stated in https://doc.nuxeo.com/corg/java-code-style/#javadoc.
Especially on tests.